### PR TITLE
Limit width of Grafana Mimir logo

### PR DESCRIPTION
**Pull request by osg-grafana**
The logo takes up too much of @bboreham's screen. How about 400px?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Grafana Mimir
 
-<p align="center"><img src="images/logo.png" alt="Grafana Mimir logo"></p>
+<p align="center"><img src="images/logo.png" alt="Grafana Mimir logo" width="400"></p>
```
Comment (pracucci) [inline README.md:3]: I think 400px width is still a lot, but in any case we should use max-width and not an explicit width to make it scale fine on small screens (eg. mobile).

```suggestion
<p align="center"><img src="images/logo.png" alt="Grafana Mimir logo" style="max-width: 400px;"></p>
```

Comment (colega) [reply README.md:3]: Does style tag work here? I couldn't find any docs on the internet.

Comment (pracucci) [reply README.md:3]: Style tag applies to any HTML visual element, including ``.

Comment (colega) [reply README.md:3]: By _here_ I meant on GitHub's markdown :)

`style` tag seems to be stripped out: https://gist.github.com/colega/88fffae30d9d26eefb8a3ebd13041c3f

## `width="20"`




## `style="max-width: 20px;"`

Comment (colega) [reply README.md:3]: BTW notice that even though the first one is `width="20"`, Github automatically adds `style="max-width: 100%"` so it won't break the layout.

**Review by colega — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pracucci) [reply README.md:3]: Ok, let's try!

Comment (osg-grafana) [reply README.md:3]: TIL :)

**Pull request merged**